<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SPDS_ClientClass</class>
 <widget class="QMainWindow" name="SPDS_ClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1375</width>
    <height>890</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1375</width>
    <height>890</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1375</width>
    <height>890</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SPDS_Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="leftMenuWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>343</width>
      <height>890</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>275</width>
      <height>890</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>890</height>
     </size>
    </property>
    <widget class="QLabel" name="logo_txt">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>166</width>
       <height>66</height>
      </rect>
     </property>
     <property name="text">
      <string>--------------------</string>
     </property>
    </widget>
    <widget class="QTreeWidget" name="leftMenuTreeWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>321</width>
       <height>731</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="login">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>830</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="upMenuWidget" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>1041</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QPushButton" name="close">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>20</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="maximize">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>20</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="minimize">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>20</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="serverConfigButton">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>20</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="cameraLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>260</y>
      <width>600</width>
      <height>360</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="mainWidget" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>1031</width>
      <height>821</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>serverConfigButton</tabstop>
  <tabstop>minimize</tabstop>
  <tabstop>maximize</tabstop>
  <tabstop>close</tabstop>
  <tabstop>leftMenuTreeWidget</tabstop>
  <tabstop>login</tabstop>
 </tabstops>
 <resources>
  <include location="SPDS_Client.qrc"/>
 </resources>
 <connections/>
</ui>
